zookeeper-server/src/main/java/org/apache/zookeeper/server/quorum/LearnerHandler.java

```diff
@@ -82,6 +84,16 @@ long getSid(){
         return sid;
     }
 
+    /**
+     * server identifier used in reporting metrics
+     */
+    private String reportId = null;
```

Comment:
The reportId was added internally to reduce the metric keys because of the large amount of observers we have, we use -1 id for all observers. 

Let's remove it from this PR and use sid directly to make it more straightforward, I don't think this is useful for the community.

Reply:
done